powershell/VstsTaskSdk/InputFunctions.ps1

```diff
@@ -114,7 +114,8 @@ function Set-TaskVariable {
     param(
         [Parameter(Mandatory = $true)]
         [string]$Name,
-        [string]$Value)
+        [string]$Value,
+        [switch]$Secret)
```

Comment:
I updated the signature here. The logging command was updated already in the previous merge.

The write-addbuildtag logging command was also added in the previous merge.

@TingluoHuang I tested them E2E and it seems to be correct. In case you want to glance them over:
https://github.com/Microsoft/vsts-task-lib/blob/master/powershell/VstsTaskSdk/LoggingCommandFunctions.ps1
ctrl+f: `write-addbuildtag`
ctrl+f: `write-setvariable`